.github/workflows/notify-docs-team.yml

```diff
@@ -47,7 +47,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "<${{ github.event.pull_request.user.html_url }}|${{ github.event.pull_request.user.login }}> has ready for review <${{ github.event.pull_request.html_url }}|TF PR ${{ github.event.pull_request.number }}>"
+                    "text": "<${{ github.event.pull_request.user.html_url }}|${{ github.event.pull_request.user.login }}> has ready for review <${{ github.event.pull_request.html_url }}|TF PR ${{ github.event.pull_request.number }}>\n${{ github.event.pull_request.title }}"
```

Comment:
In the past we got malicious users trying to inject code via the PR title so I think you should not use it in this automation

Reply:
ok, i'll remove